compiler/circle-mpqsolver/CMakeLists.txt

```
@@ -9,6 +9,8 @@ target_link_libraries(circle-mpqsolver safemain)
 target_link_libraries(circle-mpqsolver luci_lang)
 target_link_libraries(circle-mpqsolver luci_service)
 target_link_libraries(circle-mpqsolver luci_pass)
+target_link_libraries(circle-mpqsolver luci_interpreter)
+target_link_libraries(circle-mpqsolver dio_hdf5)
 
```

Nor for this PR, but you may add `nncc_common` library and resolve compile warnings.

```suggestion
target_link_libraries(circle-mpqsolver nncc_common)

```

---

ok.